packages/demos/src/start-server.ts

```diff
@@ -12,18 +13,34 @@ export interface AppRendererOptions {
 export interface AppRendererResult {
   html: string;
   serializedStates?: string;
+  stylesheetsForSsr?: Map<string, Css>;
   urlsForHydration?: Set<string>;
 }
 
 export type AppRenderer = (
   options: AppRendererOptions
 ) => Promise<AppRendererResult>;
 
+function createStylesheetLink({href, media = 'all'}: Css): string {
+  return `<link href="${href}" media="${media}" rel="stylesheet" />`;
+}
+
+function createStylesheetLinks(stylesheets: Css[]): string {
+  return stylesheets.map(createStylesheetLink).join('\n');
+}
+
 function createDocumentHtml(
   bodyHtml: string,
-  serializedStates?: string,
-  urlsForHydration?: Set<string>
+  {
+    serializedStates,
+    stylesheetsForSsr,
+    urlsForHydration
+  }: Partial<AppRendererResult> = {}
 ): string {
+  const stylesheetLinks = stylesheetsForSsr
+    ? createStylesheetLinks(Array.from(stylesheetsForSsr.values()))
```

Comment:
with the above note the call could be simplified and the conditional removed

Reply:
I thought about this, but I kind of like the symmetry with `serializedStatesScript` and `urlsForHydrationScript` below. Furthermore, I'm not so fond (anymore) of helper functions that receive a single parameter, that is then made optional and has the undefined guard inside, instead of just not calling this function from the outside.

Reply:
🤔 Or, we could of course go the other way and also create functions for the other two cases.

Reply:
It's ok for me.